cinema/serializers.py

```diff
@@ -22,3 +27,59 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class ActorSerializer(serializers.Serializer):
```

Comment:
you can use `ModelSerializer` here

Reply:
I agree, but we only got this "ModelSerializer" information for the next day's topic,
during this task "serializer. Serializer" was relevant